src/components/pages/books/books.css

```diff
@@ -185,3 +185,41 @@
 .addBtn:hover {
   cursor: pointer;
 }
+
+@media screen and (max-width: 768px) {
+  .book-section-one {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .book-title {
+    text-align: center;
+  }
+
+  .book-changes {
```

Comment:
| Zeplin's Design | @AleWaweru Design|
|-|-|
| ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/dc9e1356-d2c4-4b0b-9998-0b0115a81ef3) | ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/e2d254eb-cfb8-49b5-b616-8f3aee2ce59f) |

- [ ] It would be nice if you follow the project requirements and add the correct background color for your app to make your app look similar to the Zeplin's design to make your app look professional and attractive :+1:

Reply:
done